concent-vm/Vagrantfile

```diff
@@ -0,0 +1,87 @@
+Vagrant.configure("2") do |config|
+
+    unless Vagrant.has_plugin?("vagrant-vbguest")
+        puts "Please install `vagrant-vbguest` plugin before execute `vagrant up`."  +
+             "\nRemember to install it in other directory then concent-vm/"
```

Comment:
`then` -> `than`.

Also, I think it's better to put the `\n` at the end of the previous line.

Reply:
Done.